scripts/cleanup_indexes.py

```diff
@@ -0,0 +1,23 @@
+import logging
+import sys
+from tests.util import cleanup_indexes
+
+
+def main():
+    logging.basicConfig(level=logging.INFO)
+    logger = logging.getLogger(__name__)
+
+    if len(sys.argv) != 2:
+        logger.info("Usage: python scripts/cleanup_indexes.py <testrun_uid>")
+        sys.exit(1)
+
+    testrun_uid = sys.argv[1]
```

Comment:
Please promise me that if we're every adding even more one parameter, we're switching to `argparse` 😄